plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -47,6 +50,8 @@
 
     protected abstract SqlExecutor onRemoteDatabase();
 
+    protected Duration ruleRefreshPeriod = Duration.succinctDuration(1, MILLISECONDS);
```

Comment:
final

(it mustn't change)

Reply:
nit: static

Reply:
I'll make it final static for now until we know we'd need the flexibility (I don't see it's needed today).